### PR TITLE
Link to the impressions page front and center on the homepage

### PR DESCRIPTION
**Pull request by ghickman**

### CODE REVIEW

File: pages/index.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ PyCon UK is the annual gathering of the UK Python community and its friends
 from around the world. This year, the conference is in Cardiff, from Thursday
 15th to Monday 19th September.
 
+[Click here](/news/20160920-impressions/) to read how the attendees felt the conference went.
```
Comment (meshy) [inline pages/index.md]: How about 

`Read about [how the attendees felt the conference went](/news/20160920-impressions/).`

?

Comment (meshy) [reply pages/index.md]: (Might be slightly better for screen readers...)

Comment (ghickman) [reply pages/index.md]: Fixed in d09a59a

**Comment by meshy** [comment]
:+1:

File: pages/index.md (hunk 1 above)
Comment (meatballs) [reply pages/index.md]: Or
`Read the [thoughts, impressions and views of the conference from some of its attendees](/news/20160920-impressions/).`

Comment (ghickman) [reply pages/index.md]: Fixed in 9c84468

**Pull request merged**